dali_mqtt_daemon.py

```diff
@@ -216,9 +327,14 @@ def on_message_brightness_cmd(mqtt_client, data_object, msg):
         msg.topic,
     ).group(1)
     try:
-        if light not in data_object["all_lamps"]:
-            raise KeyError
-        lamp_object = data_object["all_lamps"][light]
+        if 'group_' in light:
```

Comment:
We need to add documentation explaining this special configuration

Reply:
added short comment in the code